nextflow.config

```diff
@@ -0,0 +1,87 @@
+manifest {
+    name = 'SNP&SEQ Run folder QC pipeline'
+    author = 'SNP&SEQ Bioinformatics Team; Mahesh Binzer-Panchal'
+    homePage = ''
+    description = 'A Nextflow run folder QC pipeline for SciLifeLab SNP&SEQ platform'
+    mainScript = 'main.nf'
+    nextflowVersion = '>=19.10.0'
+    version = '1.0'
+}
+
+profiles {
+
+    debug { process.beforeScript = 'env'}
+
+    dev {
```

Comment:
Error strategy should be set to ignore for `get_QC_thresholds` in `dev` profile too.

Reply:
Combined process specific configurations into a file and used includeConfig